WooCommerce/UITestsFoundation/Screens/Payments/PaymentsScreen.swift

```diff
@@ -90,7 +90,7 @@ public final class PaymentsScreen: ScreenObject {
 
     @discardableResult
     public func verifyPaymentsScreenLoaded() throws -> PaymentsScreen {
-        XCTAssertTrue(isLoaded)
+        XCTAssertTrue(collectPaymentButton.waitForExistence(timeout: 15))
```

Comment:
WDYT of:

```suggestion
        collectPaymentButton.waitForExistence(timeout: 15)
        XCTAssertTrue(isLoaded)
```

This way we have both extended wait, and a full screen check.

Reply:
yup we can do that too, i'll commit the suggestion